src/seedu/addressbook/ui/Formatter.java

```diff
@@ -0,0 +1,52 @@
+package seedu.addressbook.ui;
+
+import java.util.List;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+/**
+ * Formatter will have public methods used by TextUI to format text.
+ * @author unusep
+ *
+ */
+public class Formatter {
+
+    /** Format of indexed list item */
+    private static final String MESSAGE_INDEXED_LIST_ITEM = "\t%1$d. %2$s";
+
+    /** Offset required to convert between 1-indexing and 0-indexing.  */
+    public static final int DISPLAYED_INDEX_OFFSET = 1;
+    
+	/**
+	 * formatPersons takes in a list of persons and a list to store the formattedPersons,
+	 * and stores all a a list of all formatted persons in formattedPersons
+	 * @param persons
```

Comment:
Normally, you should give a description of the `@param` or the `@return` value. If you have nothing more to add, probably the comment line can be deleted as well (because it's not adding any value)?